mods/ra/rules/civilian.yaml

```diff
@@ -538,6 +538,7 @@ OILB:
 		Types: building
 	CapturableProgressBar:
 	CapturableProgressBlink:
+		Sound: gun27.aud
```

Comment:
Maybe keep this but use a more appropriate beep sound like in Generals?

Reply:
It changes the balance, we can experiment later in map mods